tests/modeladmin/tests.py

```diff
@@ -92,6 +92,20 @@ class BandAdmin(ModelAdmin):
         ma = BandAdmin(Band, self.site)
         self.assertTrue(ma.lookup_allowed('name__nonexistent', 'test_value'))
 
+    def test_lookup_allowed_reverse_onetoone(self):
+        class EmployeeProfileAdmin(ModelAdmin):
+            list_filter = ['employee__employeeinfo__description']
+
+        ma = EmployeeProfileAdmin(EmployeeProfile, self.site)
+        self.assertTrue(ma.lookup_allowed('employee__employeeinfo__description', 'test_value'))
```

Comment:
Use `assertIs(... True)` since `assertTrue()` passes for anything `bool(x) == True`.